### PR TITLE
Add `vip whoami` command

### PR DESCRIPTION
**Pull request by mjangda**
To help identify the currently logged in user. This can help with debugging login issues and during development.

## Steps to Test

1. Check out PR.
1. Run `npm run build`
1. Run `./dist/bin/vip-whoami.js`

```
❯ ./dist/bin/vip-whoami.js
- Howdy VIP User!
- Your user ID is 12345
```

For VIP users, there is an additional note: `- Your account has VIP Staff permissions`.

### CODE REVIEW

File: src/lib/api/user.js [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+export async function getCurrentUserInfo() {
+	const api = await API();
+
+	let response = {};
+	try {
+		response = await api.query( { query: QUERY_CURRENT_USER } );
+	} catch ( err ) {
+		throw err;
+	}
```
Comment (nickdaugherty) [inline src/lib/api/user.js]: Since we're not doing anything with the error (except re-throwing), we could omit the try/catch.

**Review by nickdaugherty — APPROVED** [review]
This is awesome! I left a non-blocking comment about a try/catch, but 🚢

File: src/lib/api/user.js (hunk 1 above)
Comment (mjangda) [reply src/lib/api/user.js]: Good idea; ff0a619

**Comment by simonwheatley** [comment]
Thank you for building a feature just for me, @mjangda 🥰

**Pull request merged**